docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -19,117 +19,137 @@ Custom add-on packs allow you to deploy Kubernetes applications in clusters and
 
 - Enterprises can add proprietary Kubernetes applications to a custom add-on pack.
 
-In this tutorial, you will create a custom add-on pack to package a sample Kubernetes application, [Hello Universe](https://github.com/spectrocloud/hello-universe#hello-universe), and deploy that application to a cluster. You will learn to create the pack in two ways, using manifest files and Helm charts. 
+In this tutorial, you will create a custom add-on pack to package a sample Kubernetes application, [Hello Universe](https://github.com/spectrocloud/hello-universe#hello-universe), and deploy the application to a cluster. You will learn to create the pack in two ways: using manifest files and Helm charts. 
 
-After defining the custom pack, you will set up a registry server, publish the pack to that registry, and configure the registry server in Palette. Lastly, you will create a cluster profile that contains your custom pack and apply the profile to a cluster using either Palette or Terraform.  
+After defining the custom pack, you will set up a new registry server or leverage an existing Open Container Initiative (OCI) registry. Then, you will publish the pack to the registry and configure the registry server in Palette. Lastly, you will create a cluster profile that contains your custom pack and apply the profile to a cluster using either Palette or Terraform. 
```

Comment:
Should these steps be in a numbered list, since they are numbered on the diagram?

Reply:
So, these steps don't match exactly the steps displayed in the diagram. For example, we don't have the registry set up present in the diagram, but we have it in the text. I added the 'sequential' word to improve the understanding.